Dockerfile

```diff
@@ -28,19 +29,28 @@ SHELL ["/bin/bash", "-c"]
 # Install commands used in our scripts and standard present on a clean ubuntu
 # installation and setup a user with sudo priviledges
 RUN apt-get update -qq && \
-    apt-get install -qq --assume-yes apt-transport-https apt-utils ca-certificates curl dbus dialog git lsb-release sudo tzdata wget && \
+    apt-get install -qq --assume-yes --no-install-recommends apt-transport-https apt-utils ca-certificates curl dbus dialog git lsb-release openssh-client sudo tzdata wget > /dev/null && \
     # Add amigo user
-    adduser --disabled-password --gecos "" $USER && \
-    adduser $USER sudo && \
-    echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+    adduser -u 1000 --disabled-password --gecos "" $USER && \
+    usermod -aG sudo $USER && \
+    usermod -aG adm $USER && \
+    echo "%sudo ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers.d/"$USER"
 
 # Setup the current user and its home directory
 USER "$USER"
 WORKDIR /home/"$USER"
 
+ADD installer/bootstrap.bash ./bootstrap.bash
+
+ADD ci/ssh-merge-known_hosts.py ./ssh-merge-known_hosts.py
```

Comment:
As this file is not being used for docker build, there is no issue of adding it here. It automatically gets cloned in tue-env.

Reply:
True